docker/rhel/Dockerfile

```diff
@@ -22,13 +22,16 @@ RUN <<EOF
 pkgs=()
 pkgs+=(ca-certificates)  # Enable TLS verification for HTTPS connections by providing trusted root certificates.
 pkgs+=(cmake)            # Required build tool.
+pkgs+=(file)             # Required packaging tool.
 pkgs+=(git)              # Required build tool.
 pkgs+=(gpg)              # Dependency for tools requiring signing or encrypting/decrypting.
 pkgs+=(jq)               # Pretty printing.
 pkgs+=(libstdc++-static) # Required to statically link libraries into rippled.
 pkgs+=(ninja-build)      # Required build tool.
 pkgs+=(perl-FindBin)     # Required to compile OpenSSL.
 pkgs+=(python3-pip)      # Package manager for Python applications.
+pkgs+=(rpm-build)        # Required packaging tool.
+pkgs+=(rpmdevtools)      # Required packaging tool.
 pkgs+=(wget)             # Required build tool.
 dnf install -y --setopt=tsflags=nodocs "${pkgs[@]}"
```

Comment:
Please also add

```
dnf clean all
rm -rf /var/cache/dnf/*
```

(and a similar change in Debian and Ubuntu already suggested in https://github.com/XRPLF/ci/pull/4#discussion_r2180089456 )